jaseci_kit/jaseci_kit/modules/t5_sum/README.md

```diff
@@ -0,0 +1,188 @@
+# **Summarization(`t5_sum`)**
+
+Module `t5_sum` uses the hugging face T5 transformer model to extract summary from text.
```

Comment:
transformer model to provide abstractive summary from text.

Reply:
done